content/swiftui/concepts/notifications/notifications.md

```diff
@@ -0,0 +1,140 @@
+UserNotifications framework helps to schedule and manage notifications in both cases whether they are local or remote (push) notifications.
+
+## Local Notifications
+
+- `Definition`: Local notifications are created and scheduled within the app itself, without any need for an external server. They are ideal for reminders, calendar events, or any other notifications that are triggered by the app based on user actions or predefined schedules.
+
+- `Scheduling`: The developer sets up the timing and content of these notifications. They can be scheduled to appear at a specific time or in response to a particular event within the app.
+
+- `User permissions`: Though less intrusive than remote notifications, local notifications still require user permission. The app must request and obtain permission to send notifications to the user.
+
+- `Customization`: Local notifications can be customized with various options such as sound, alerts, or badges. They can also be configured to trigger certain actions when interacted with.
+
+- `Implementation in SwiftUI`: Implementing local notifications in SwiftUI involves using the UNUserNotificationCenter to manage notification content, request permission, and schedule the notification delivery.
+
+Code example below creates a basic notification with a time-based trigger:
+
+```swift
+import UserNotifications
+
+func scheduleTimeBasedNotification() {
+  // 1. Request permission to display alerts and play sounds.
+  UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound]) { granted, error in
+    if granted {
+        print("Permission granted")
+    } else if let error = error {
+        print(error.localizedDescription)
+    }
+}
+
+  // 2. Create the content for the notification
+  let content = UNMutableNotificationContent()
+  content.title = "Reminder"
+  content.body = "Don't forget to check the app!"
+  content.sound = UNNotificationSound.default
+
+  // 3. Set up a trigger for the notification
+  // For example, 10 seconds from now
+  let trigger = UNTimeIntervalNotificationTrigger(timeInterval: (10), repeats: false)
+
+  // 4. Create the request
+  let request = UNNotificationRequest(identifier: UUID().uuidString, content: content, trigger: trigger)
+
+  // 5. Add the request to the notification center
+  UNUserNotificationCenter.current().add(request) { error in
+    if let error = error {
+        print(error.localizedDescription)
+    } else {
+        print("Notification scheduled")
+    }
+  }
+}
+```
+![Simulator Screenshot - iPhone 15 Pro - 2023-12-20 at 16 41 31](https://github.com/arirays/docs/assets/51388551/ded8763a-245f-4ea3-af16-d680b7d0bec4)
```

Comment:
can you add this image in the media and make this link like this:
https://raw.githubusercontent.com/Codecademy/docs/main/media/background-position-example-1.png